charts/prometheus-stackdriver-exporter/values.yaml

```diff
@@ -68,8 +68,11 @@ stackdriver:
   metrics:
     # The prefixes to gather metrics for, we default to just CPU metrics.
     typePrefixes: 'compute.googleapis.com/instance/cpu'
-    # The monitoring filters to specify resources, metric types, group definitions, and time series.
-    filters: ""
+    # The filters to refine the metrics query by using Filter objects that Google provides.
+    # Filter objects: project, group.id, resource.type, resource.labels.[KEY], metric.type, metric.labels.[KEY]
+    # https://cloud.google.com/monitoring/api/v3/filters
+    filters: []
```

Comment:
this would be a breaking change for users already using the value.
therfore new major version would need to be created.

Reply:
Good point. I've adjusted the major version @monotek 👍